example/m3db-cluster.yaml

```diff
@@ -214,13 +214,15 @@ metadata:
   name: m3db-cluster
 spec:
   image: quay.io/m3/m3dbnode:latest
-  replicationFactor: 1
-  numberOfShards: 8
+  replicationFactor: 3
+  numberOfShards: 256
```

Comment:
Curious where this number came from?

Reply:
My understanding is that it will speed up bootstraps since we can do more shards in parallel